File: 215. Kth Largest Element in an Array/kthLargestElement.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        nums = sorted(nums)
        if len(nums)>=2:
            k = int(k)
            return nums[-k]

        else:
            nums = ''.join(map(str, nums))
            return nums
        

File: 215. Kth Largest Element in an Array/kthLargestElement.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        //Since the array can contain duplicates, all we need to do
        //is sort the nums array and return the index of the kth element
        Arrays.sort(nums);
        return nums[nums.length -k];
        
    }
}
